src/main/scala/nl/knaw/dans/lib/logging/servlet/ResponseLogFormatter.scala

```diff
@@ -34,6 +34,9 @@ trait ResponseLogFormatter {
     val status = actionResult.status
     val formattedAuthHeaders = formatResponseHeaders(getHeaderMap(response)).makeString
     val formattedActionHeaders = formatActionHeaders(actionResult.headers).makeString
+    // review comment:
+    // somewhere I stumbled over the same header by the response headers and action headers
+    // don't remember or did not test what was actually sent to the client in those occasions
```

Comment:
not sure what you mean here. If you find it again, it would be great if we could incorporate any such thing before releasing it!